Release automation is not working with Mozart and vendor-bin changes

Ref https://github.com/nextcloud-releases/spreed/actions/runs/2972967840

> # Install composer dependencies
> Run cd spreed
> Installing dependencies from lock file
> Verifying lock file contents can be installed on current platform.
> Package operations: 1 install, 0 updates, 0 removals
>   - Downloading firebase/php-jwt (v6.3.0)
>   - Installing firebase/php-jwt (v6.3.0): Extracting archive
> Generating optimized autoload files
> 
> Error: Command "bin" is not defined.
>                                  
>   Command "bin" is not defined.  
                                 

Script @composer bin all install --ansi handling the post-install-cmd event returned with error code 1
Error: Process completed with exit code 1.


Blind guess:
https://github.com/nextcloud/spreed/blob/master/.github/workflows/appstore-build-publish.yml#L84
needs to install without `--no-dev`

--- a/composer.json
+++ b/composer.json
@@ -42,11 +42,11 @@
 		"test": "cd tests/php/ && phpunit -c phpunit.xml"
 	},
 	"require-dev": {
-		"bamarni/composer-bin-plugin": "^1.8",
 		"christophwurst/nextcloud": "dev-master",
 		"roave/security-advisories": "dev-latest"
 	},
 	"require": {
+		"bamarni/composer-bin-plugin": "^1.8",
 		"firebase/php-jwt": "^6.3"
 	},
 	"extra": {

